src/content/development/_index.en.md

```diff
@@ -18,3 +18,14 @@ pre = "<b>4. </b>"
 * [Working with Shipyard](shipyard)
   * [Adding Shipyard to a Project](shipyard/first-time)
   * [Advanced Features](shipyard/advanced)
+
+### Research notice
```

Comment:
Markdownlint is failing because you need a blank line after the section header.

> src/content/development/_index.en.md: 22: MD022/blanks-around-headings/blanks-around-headers Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below] [Context: "### Research notice"]

Reply:
```suggestion
### Research notice

```